### ISSUE TITLE
No pretrained model provided.

### ISSUE THREAD

**Issue by Charlie-lusie** [overfitover/pixor_pytorch#7]
I'm training it by myself, but I also want to try pre-trained models. Hope you can release the model, Thanks.

**Comment by overfitover** [comment]
链接: https://pan.baidu.com/s/1GI0hNdl2zJprCd3tO1s7_g 
提取码: kkb8 
The model may be changed. You can try.

**Comment by Charlie-lusie** [comment]
Thanks for your weights.

But I found that the structure of your weights is different from what written in the code. I changed the net structure and loaded the weights, then found that no results for any input data. Do u know the reason?(maybe you send the wrong weights?)

________________________________
发件人: overfitover <notifications@github.com>
发送时间: Wednesday, April 17, 2019 6:59:43 PM
收件人: overfitover/pixor_pytorch
抄送: Shuai Yang; Author
主题: Re: [overfitover/pixor_pytorch] No pretrained model provided. (#7)


链接: https://pan.baidu.com/s/1GI0hNdl2zJprCd3tO1s7_g
提取码: kkb8
The model may be changed. You can try.

―

**Comment by overfitover** [comment]
@Charlie-lusie 
The model I trained before was deleted. This is the result of my friend's training, he may have modified the network structure. You can try to train yourself. In the case of nan in the training process, you can use small data to verify the validity of the network structure, and try to modify the loss function to correct it.

**Comment by Charlie-lusie** [comment]
I got it, thank u anyway 😊

________________________________
发件人: overfitover <notifications@github.com>
发送时间: Friday, April 19, 2019 6:15:56 PM
收件人: overfitover/pixor_pytorch
抄送: Shuai Yang; Mention
主题: Re: [overfitover/pixor_pytorch] No pretrained model provided. (#7)


@Charlie-lusie<https://github.com/Charlie-lusie>
The model I trained before was deleted. This is the result of my friend's training, he may have modified the network structure. You can try to train yourself. In the case of nan in the training process, you can use small data to verify the validity of the network structure, and try to modify the loss function to correct it.

**Comment by andrearama** [comment]
Hi! 
Did you manage to get a trained model with the current implementation? @Charlie-lusie @overfitover 

Thanks for your time :)

**Comment by Charlie-lusie** [comment]
> Hi!
> Did you manage to get a trained model with the current implementation? @Charlie-lusie @overfitover
> 
> Thanks for your time :)

I tried to use the trained model provided by @overfitover by changing the code. And the result is bad. Then I trained it myself,  but the results hadn't changed much.

**Comment by andrearama** [comment]
Thanks for the reply

Agree with you, happened the some to me.
Delving more in the code I think I've noticed some bugs/irregularities. I will try to fix them and push
